src/MCPClient/lib/clientScripts/trimVerifyChecksums.py

```diff
@@ -64,21 +64,21 @@ def callWithException(exception):
         try:
             tree = etree.parse(xmlFilePath)
             root = tree.getroot()
-    
+
             #extension = root.find("Document/Extension").text
             xmlMD5 = root.find("Document/MD5").text
         except:
-            print >>sys.stderr, "Error parsing: ", xmlFilePath 
+            print >>sys.stderr, "Error parsing: ", xmlFilePath
             exitCode += 1
             continue
         #if extension.lower() != file[i+1:].lower():
-        #    print >>sys.stderr, "Warning, extension mismatch(file/xml): ", file[:i], extension , file[i+1:] 
-        
-        objectMD5 = md5_for_file(filePath)
-        
+        #    print >>sys.stderr, "Warning, extension mismatch(file/xml): ", file[:i], extension , file[i+1:]
```

Comment:
Since you're doing cleanup anyway, dead code like this can be removed if it's not still relevant.